packages/flutter/lib/src/rendering/object.dart

```diff
@@ -4736,43 +4996,47 @@ class _RootSemanticsFragment extends _InterestingSemanticsFragment {
   });
 
   @override
-  void compileChildren({
-    Rect? parentSemanticsClipRect,
-    Rect? parentPaintClipRect,
+  bool compileSemanticsNodes({
```

Comment:
Why does this override change the return type? I also couldn't find any place that uses this return value - and it seems to always return true?